### PR TITLE
Add sync modes to Postgres SQL destination plugin.

### PR DESCRIPTION
**Pull request by pauloouriques**
- Minor fix to redundant sync modes check on sqlite plugin.

### CODE REVIEW

File: plugins/@grouparoo/postgres/package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "@types/pg": "*",
     "actionhero": "25.0.16",
     "csv-parse": "4.15.4",
+    "fs-extra": "10.0.0",
```
Comment (evantahler) [inline plugins/@grouparoo/postgres/package.json:45]: Doesn't look like this is used?

**Review by evantahler — CHANGES_REQUESTED** [review]
Looks good except for that dependency

File: plugins/@grouparoo/postgres/package.json (hunk 1 above)
Comment (pauloouriques) [reply plugins/@grouparoo/postgres/package.json:45]: It's used [here](https://github.com/grouparoo/grouparoo/blob/2426f644938144c9a5399388268afa105b8df4ba/plugins/%40grouparoo/postgres/__tests__/integration/cli-generate-from.ts#L2)

Comment (evantahler) [reply plugins/@grouparoo/postgres/package.json:45]: whoa!  So we had just been lucky that it was brought in as a transient dependency (probably via @grouparoo/core) until now.  Good find!

**Review by evantahler — APPROVED** [review]

**Pull request merged**